### PR TITLE
pojo to timestamp field

### PR DESCRIPTION
**Pull request by flashmouse**
i add a new annotation for timestamp so that we can set point time and precision attribute when use addFieldsFromPOJO method,and also can reslove UnableToParseException when use this method with a pojo class has field which type is Instant

### CODE REVIEW

**Comment by majst01** [comment]
Hi @flashmouse,

thanks for your contribution, would you please add
- a unit test covering this new functionality
- extend example in README.md
- add a CHANGELOG.md entry

other than that looks good.

**Comment by flashmouse** [comment]
master branch version now is 2.16-SNAPSHOT and CHANGELOG latest change is 2.15 ,should I add 2.16 entry for CHANGELOG?

**Comment by majst01** [comment]
yes of course

File: src/main/java/org/influxdb/dto/Point.java [hunk 1]
```diff
@@ -263,10 +264,11 @@ private void addFieldByAttribute(final Object pojo, final Field field, final Col
 
         TimeColumn tc = field.getAnnotation(TimeColumn.class);
         if (tc != null && Instant.class.isAssignableFrom(field.getType())) {
```
Comment (flashmouse) [inline src/main/java/org/influxdb/dto/Point.java]: what should I do if some filed has TimeColumn annotation but filed type not Instant?

**Pull request merged**